frontend/src/components/TextArea/index.tsx

```diff
@@ -48,7 +49,7 @@ export const TextArea: React.FC<TextInputProps> = ({
                     'transition-all'
                 )}
             >
-                {touched && value ? label : placeholder}
+                {value ? label : placeholder}
```

Comment:
Did you pull the latest version of the main?
There should not be any changes to the TextArea.

Reply:
dont know how this change got here. fixing